cmd/cli/cmd/typeinstance/edit.go

```diff
@@ -83,20 +85,36 @@ func editTI(ctx context.Context, opts editOptions, w io.Writer) error {
 func typeInstanceViaEditor(ctx context.Context, cli client.Hub, tiID string) ([]gqllocalapi.UpdateTypeInstancesInput, error) {
 	out, err := cli.FindTypeInstance(ctx, tiID)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "while finding TypeInstance")
 	}
 	if out == nil {
 		return nil, fmt.Errorf("TypeInstance %s not found", tiID)
 	}
 
-	rawInput, err := yaml.Marshal(mapTypeInstanceToUpdateType(out))
+	backendData, err := typeinstance.NewStorageBackendData(ctx, cli, out)
+	if err != nil {
+		return nil, errors.Wrap(err, "while fetching storage backend data")
+	}
+
+	updateTI := mapTypeInstanceToUpdateType(out)
+
+	var valueData string
+	if backendData != nil && !backendData.AcceptValue {
```

Comment:
I see that it duplicates the logic from the `setTypeInstanceDataForMarshaling` - you can give it a second thought whether it will be ok to unify it or not. I briefly check that opt, but it didn't play well with the usage in `get.go`

Reply:
Yes, that was my first thought also but then it turned out that it is not clear how to unify it better. Let's keep it for now.